CHANGES.md

```diff
@@ -15,6 +15,7 @@ Language grammar improvements:
 - enh(php) Add `mixed` to list of keywords (#2997) [Ayesh][]
 - enh(php) Add support binary, octal, hex and scientific numerals with underscore separator support (#2997) [Ayesh][]
 - enh(php) Add support for Enums (#3004) [Ayesh][]
+- enh(ecmascript) Add built-in types [Vaibhav Chanana][]
```

Comment:
Please also finish this link by adding it's target on line ~25ish.

Reply:
Done

Reply:
The link is already there down in the document. Do we need to add it again?

Reply:
For every release.  The document is split at release time - so only the current release exists.